### PR TITLE
Handle unicode characters in feedback output when exporting

### PR DESCRIPTION
**Pull request by hillaryj**
Feedback export was failing due to an unhandled Unicode character in the output.

## Additions

- Adds encoding for the referrer URL, which may contain a unicode checkbox.

## Testing

1. Go to your local admin in the feedback: http://localhost:8000/django-admin/v1/feedback/
1. Select all rows.
1. Click "Select all 87233 feedbacks" in the table header area
1. Select the export option in the drop down and click "Go".

You should get a message that a CSV file has been sent and not a 500 error.

## Notes

- In the python3 future, we'll be able to specify the coding for the csv writer instead.

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] Any *change* in functionality is tested
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

**Comment by richaagarwal** [comment]
@hillaryj I don't see any encoding changes in the diff, just that you're checking if the string is `None` or not.  Should the commit message and PR title/description be updated to be in sync with the change?

File: cfgov/v1/models/base.py [hunk 1]
```diff
@@ -473,6 +473,8 @@ def assemble_csv(self, queryset):
         for feedback in queryset:
             feedback.submitted_on = "{}".format(feedback.submitted_on.date())
             feedback.comment = feedback.comment.encode('utf-8')
+            if feedback.referrer is not None:
```
Comment (richaagarwal) [inline cfgov/v1/models/base.py]: Minor thing but this could also be simplified to simply `if feedback.referrer:`

**Comment by hillaryj** [comment]
Encoding the `feedback.referrer` is new, as a checkbox shows up in the URL in some saved feedbacks.

**Comment by richaagarwal** [comment]
Oh, sorry, somehow missed that 2nd line when reviewing just now.

**Pull request merged**